crates/driver/src/domain/eth/mod.rs

```diff
@@ -170,6 +170,24 @@ impl From<Address> for H160 {
     }
 }
 
+// TODO This type should be more complex and ensure that the private key is
+// valid, use secp256k1?
+/// A private key that can be used to sign transactions.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct PrivateKey(pub H256);
```

Comment:
Why not use `ethcontract::PrivateKey`?

Reply:
Good point https://github.com/cowprotocol/services/pull/1124/commits/fc33c8cb1bc4b8dc8a5ada1cafdccdce5958caf5